tensorrt_llm/_torch/speculative/dflash_hybrid_attn.py

```
@@ -0,0 +1,478 @@
+# Reused across steps so CUDA graph capture sees stable addresses; keyed by
+# (B, NKV, S, R, D, device). Bounded: shapes are fixed per serving config.
+_PARTIAL_BUFS = {}
+
+
+def _get_partial_bufs(B, NKV, S, R, D, device):
+    key = (B, NKV, S, R, D, device)
+    bufs = _PARTIAL_BUFS.get(key)
+    if bufs is None:
+        acc = torch.empty(B * NKV * S * R * D, dtype=torch.float32, device=device)
+        m = torch.empty(B * NKV * S * R, dtype=torch.float32, device=device)
+        l = torch.empty(B * NKV * S * R, dtype=torch.float32, device=device)
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)